packages/v4/src/content/design-guidelines/usage-and-behavior/lists-and-tables/lists-and-tables.md

```diff
@@ -8,20 +8,33 @@ showTitle: true
 PatternFly offers two components for displaying large data sets: [data list](/documentation/react/components/datalist/) and [table](/documentation/react/components/table/). While they satisfy similar use cases, choosing the correct component to use in your design will be dependent on the type of data you need to display.
 
 Use [data lists](#data-lists) when:
-* A flexible layout is more important than arranging information in a grid
-* You want to include active content like a chart
-* Content displayed may vary between rows
+* A flexible layout is more important than arranging information in a grid.
+* You want to include active content like a chart.
+* Content displayed may vary between rows.
 
 Use [tables](#tables) when:
-* Users will want to consume data as a grid (i.e. structured rows and columns)
-* You want column headers
+* Users will want to consume data as a grid (i.e. structured rows and columns).
+* You want column headers.
 
 ## Data lists
 PatternFly supports several variations of the data list component.
```

Comment:
I would add here "PatterFly supports two main types of data lists
1. Compact data list
2. Basic data list"

and then link both of those. Just to introduce both those types from the beginning, so that when you mention them again under "Data list capabilities" people know what you're talking about!